### ISSUE TITLE
Separate out bytea columns in database to separate tables (Policy and Model)

### ISSUE THREAD

**Issue by FollowSteph** [PathmindAI/pathmind-webapp#446]
First it's good practice to separate out blobs into a separate linked table for a number of reasons, but more importantly for us is to do so as a defensive programming method to avoid any accidental memory leaks. All it takes right now is for someone to do MODEL.asterisk() or POLICY.asterisk() and we will have almost certainly have memory and performance issues. Keep in mind that unless there is a reason most of the current code does this everywhere which is not a bad thing (it future proofs adding new columns, etc as most of the time we'll want all the columns since our database isn't that complex). 

In other words if we don't do this the next time someone tries to create a query to get a list of policies or models and just does a select * we'll be back in the same boat and won't realize it until the server starts crashing. And even then it will most likely not be obvious that this is the cause.

The effort here is to create a separate database table for the blob for the Policy.file and Model.file. I don't think it's worth it for the json progress string as we're looking to remove it shortly anyways #387 and #388.
